decidim-proposals/app/views/decidim/proposals/proposals/participatory_texts/participatory_text.html.erb

```diff
@@ -2,7 +2,7 @@
 
 <div class="row view-header">
   <div class="column medium-9">
-    <h2 class="heading2"><%= translated_attribute(@participatory_text.title) %></h2>
+    <h2 class="heading2"><%= translated_attribute(@participatory_text&.title) %></h2>
```

Comment:
I'm not sure this is the best approach, we should display something like "There are no participatory texts at the moment" instead of just a blank title.

Reply:
My colleage was saying the same thing. Thanks. I'll do that.